react/Header/NotificationsLink/NotificationsLink.js

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react';
+import BellIcon from '../../BellIcon/BellIcon';
+import styles from './NotificationsLink.less';
+import { getCookie } from './utils';
+import { AUTHENTICATED } from '../../private/authStatusTypes';
+
+class NotificationsLink extends React.Component {
+  state = {
+    isInExperiment: false
+  }
+
+  componentDidMount() {
+    const EXPERIMENT_ID = '58CC4E12-A4E0-4C0B-BD2A-681FD98FDC2E';
+    const visitorId = getCookie('JobseekerVisitorId');
+    const url = `http://experiments-api.candidate.prod.outfra.xyz/participants/${visitorId}`;
```

Comment:
This URL is not `https`. It'll bork on the site. You'll need to use cloud.seek.com.au one I think.